### ISSUE TITLE
How to generate simulated data?

### ISSUE THREAD

**Issue by athowes** [CDCgov/cfa-respnet-nowcasting-daily#19]
Hello! I am having a go at running through the simulated data analysis here.

In the README:

> Synthetic data were simulated from an SEIHRD model as in `R/Simulated Data Analysis/Functions/sim_hospital_admissions.R`, and one test simulation was used for all model runs on simulated data. The original simulated data is entirely on the daily scale: both hospital admission and report dates are available at daily resolution. Code used to process sim data are in `R/Simulated Data Analysis/process_sim_data.R`.

So to start I am trying to run `process_sim_data.R`.  This starts by calling:

https://github.com/CDCgov/cfa-respnet-nowcasting-daily/blob/c7e31e6d532c7c1e6e97427d6aaae98ca77eebe7/R/Simulated%20Data%20Analysis/process_sim_data.R#L7

However there isn't a data folder (and I guess this `.csv` would be ignored too). How should I generate `sim_data.csv`? When I try find in files for `sim_data` it doesn't show so I guess there isn't existing code in the repo to generate. Perhaps it's quite easy with `sim_seihrd` but would be good to have the code.

**Comment by jessalynnsebastian** [comment]
Yes, very good point, this code chunk should be in there. I'll see if I can still push changes directly to add it, and I can also just email you the csv I was using.

**Comment by jessalynnsebastian** [comment]
Edited the process_sim_data.R file to include code to generate the csv.
